backend/app/routers/experiment_templates.py

```diff
@@ -114,16 +209,32 @@ async def approve_experiment_template(
         )
 
     experiment_template.approved = is_approved
-    experiment_template.updated_at = datetime.utcnow()
+    experiment_template.updated_at = datetime.now(tz=timezone.utc)
 
     await ExperimentTemplate.replace(experiment_template)
     await exp_scheduler.add_image_to_build(experiment_template.id)
 
 
+@router.get("/experiment-templates/{id}/exist-experiments", response_model=bool)
```

Comment:
I am not sure about this endpoint.. I understand it might be helpful from the FE perspective, but I don't see it useful from the BE perspective. I would rather add an endpoint for listing all experiments based on a specific template (or only count). We can then use this endpoint from FE for our purposes.